tests/e2e/testcases/blockchain/configure/suite.go

```diff
@@ -0,0 +1,470 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package configure
```

Comment:
this should be in commandse2e?

Reply:
there is a tendency to set up all new test cases under tests/e2e/testcases/cmdPath/, probably we want to
move remaining ones on comandse2e so everything test case related